frameworks/OCaml/.gitignore

```diff
@@ -2,3 +2,4 @@
 **/_build
 .merlin
 .ocamlformat
+!**/bin
```

Comment:
in OCaml /bin is a common naming convention for putting executables, so removing it from the global `.gitignore`
https://dune.build/examples